metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-875
+897
```

Comment:
setup.py (which we determined can't effectively be split out into imports) has more code now.